### PR TITLE
Document Linux distribution requirements

### PR DESCRIPTION
**Pull request by domesticmouse**
Adding notes to the Desktop page detailing how the Linux desktop app is compiled, and the requirements for using the compiled binary on other systems.

### CODE REVIEW

File: src/desktop.md [hunk 1]
```diff
@@ -357,14 +357,36 @@ impact your distributable application.
 
 #### Linux
 
+The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 
+executable binary in the `bundle` directory there are two directories, `lib` which contains required `.so`
```
Comment (stuartmorgan) [inline src/desktop.md:361]: Nit: s/,/:/

File: src/desktop.md [hunk 2]
```diff
@@ -357,14 +357,36 @@ impact your distributable application.
...
+library files, and the `data` directory that contains data assets that your application relies on.
```
Comment (stuartmorgan) [inline src/desktop.md:362]: Nit: s/that/which/ for parallel structure.

**Review by stuartmorgan — APPROVED** [review]
LGTM with nits. Thanks!

File: src/desktop.md (hunk 2 above)
Comment (domesticmouse) [reply src/desktop.md:362]: Re-worded, but I'm unsure if I like the new structure. I'll let @sfshaza2 word smith my prose =)

File: src/desktop.md (hunk 1 above)
Comment (domesticmouse) [reply src/desktop.md]: Refactored the two points into a list, following the new `:` in the text.

File: src/desktop.md [hunk 3]
```diff
@@ -358,8 +358,9 @@ impact your distributable application.
 #### Linux
 
 The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 
-executable binary in the `bundle` directory there are two directories, `lib` which contains required `.so`
-library files, and the `data` directory that contains data assets that your application relies on.
+executable binary in the `bundle` directory there are two directories:
+ - `lib` which contains required `.so` library files, and 
+ - `data` which contains the data assets that your application relies on.
```
Comment (sfshaza2) [inline src/desktop.md:363]:
```suggestion
executable binary in the `bundle` directory there are two directories:
 * `lib` contains the required `.so` library files
 * `data` contains the application's data assets, such as fonts or images
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Hopefully the suggested assets are good examples. If not, feel free to replace.

File: src/desktop.md (hunk 3 above)
Comment (domesticmouse) [reply src/desktop.md]: Yup, much betterer

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**